File: src/app/gkill/api/req_res/update_kmemo_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type UpdateKmemoResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedKmemo *reps.Kmemo `json:"updated_kmemo"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`
}

File: src/client/classes/api/req_res/update-kmemo-response.ts
'use strict'

import { Kmemo } from '@/classes/datas/kmemo'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateKmemoResponse extends GkillAPIResponse {

    updated_kmemo: Kmemo

    updated_kyou: Kyou | null

    constructor() {
        super()
        this.updated_kmemo = new Kmemo()
        this.updated_kyou = null
    }

}


